### ISSUE TITLE
tracked deletions are being checked

### ISSUE THREAD

**Issue by milekpl** [jaumeortola/languagetool-msword10-addin#2]
The addon sends the deleted but tracked text. To replicate:
1. Write "test".
2. Turn on tracking changes.
3. Remove "test". Write "test" again.
4. Start the check. LT will complain about duplicate words (or about spelling mistake in "testtest", depending on where you put "test").

**Comment by milekpl** [comment]
As far as I can see, you need to use `Revisions` property, see:
https://msdn.microsoft.com/en-us/library/office/ff838481.aspx

Apparently, then you need to remove text that was deleted by checking the status of the revision:

http://www.vbaexpress.com/forum/showthread.php?45157-Who-can-explain-to-me-the-revisions-object

**Comment by milekpl** [comment]
I think there's a good workaround: if we switch off showing the tracked changes, then everything works fine. And it's not really needed during checking, so I suggest that the dialog should store the state of how tracked changes are shown, turn it off if it's on, and then restore the state. It's quite straightforward programmatically:

https://social.msdn.microsoft.com/Forums/office/en-US/c132d40d-9a5b-4baa-a4c7-2c04628d8d2b/reading-final-range-text-with-track-changes-on-in-word?forum=worddev

**Comment by jaumeortola** [comment]
The workaround works for reading the text without tracked changes, but not for making changes on the text from the dialog (the positions have to be adjusted). Anyway, I think we are close to the solution.

**Comment by milekpl** [comment]
Right, you need only to take care of subtracting the text length that was marked as deleted in Revisions.
